app/components/ucsf/Section/Section.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import styles from './Section.module.scss';
+
+const Section = ({
+  title, subtitle, body, addClass,
+}: {
+  title?: string;
+  subtitle?: string;
+  body?: string;
+  addClass?: string;
+}) => (
+  <section className={`${styles.section} ${addClass || ''}`}>
```

Comment:
Good catch! I switched on the `restrict-template-expressions` and saw that it complains about `style.className` expressions that map to valid styles. It gives the following error: `Invalid type "any" of template literal expression`. We have quite a few of these, so I left this off for now. I guess that is not typesafe as it could be an undefined element. Any thoughts on this?